src/app/manage/manage.component.css

```diff
@@ -0,0 +1,34 @@
+.mycards{
+    display: flex;
+    flex-direction: row;
+}
+.mycard{
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+}
+
+.deleted-card{
+    border: 2px solid red;
+}
+
+.active-card{
+    border: 2px solid green;
+}
+
+.card{
```

Comment:
`Bootstrap` css already has a `card` class. You can use them instead of writing your own css